truck_inspection_client/launch/truck_inspection_client.launch

```diff
@@ -20,7 +20,7 @@
     <remap from="incoming_mobility_request" to="$(optenv CARMA_MSG_NS)/incoming_mobility_request"/>
     <remap from="guidance/state" to="$(optenv CARMA_GUIDE_NS)/state"/>
 
-    <node name="truck_inspection_client" pkg="truck_inspection_client" type="truck_inspection_client">
+    <node name="truck_inspection_client" pkg="truck_inspection_client" type="truck_inspection_client"  output="screen">
```

Comment:
Don't change this to output=screen. Log files are easier to parse through when errors occur.

Reply:
Addressed: https://github.com/usdot-fhwa-stol/carma-platform/pull/1020/commits/56dd79335ca2255d024912c27d7e13719a88414d